geograypher/meshes/meshes.py

```diff
@@ -327,8 +337,8 @@ def set_texture(
             )
 
         # If IDs to labels is explicitly provided, trust that
-        if IDs_to_labels is not None:
-            # TODO should do some type checking here
+        # TODO should do some type checking here
+        if isinstance(IDs_to_labels, dict):
```

Comment:
Should maybe do something if it's not a dict and not None. But I'm not sure exactly what.